build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -98,27 +49,107 @@
   when:
     - __service_telemetry_observability_strategy in ['use_redhat', 'use_hybrid']
 
-- name: Subscribe to Elastic Cloud on Kubernetes Operator
+# undocumented procedure, used for testing updates or old deployment models
+- name: Subscribe to Prometheus Operator
   kubernetes.core.k8s:
     definition:
       apiVersion: operators.coreos.com/v1alpha1
       kind: Subscription
       metadata:
-        name: elasticsearch-eck-operator-certified
+        name: prometheus
         namespace: "{{ namespace }}"
       spec:
-        channel: stable
+        channel: beta
         installPlanApproval: Automatic
-        name: elasticsearch-eck-operator-certified
-        source: certified-operators
+        name: prometheus
+        source: community-operators
         sourceNamespace: openshift-marketplace
+  when:
+    - __service_telemetry_observability_strategy == "use_community"
+
+# documented procedure: https://infrawatch.github.io/documentation/#deploying-certificate-manager-for-openshift-operator_assembly-installing-the-core-components-of-stf
+- block:
+  - name: Create project for cert-manager for Red Hat OpenShift
+    kubernetes.core.k8s:
+      definition:
+        apiVersion: project.openshift.io/v1
+        kind: Project
+        metadata:
+          name: cert-manager-operator
+        spec:
+          finalizers:
+          - kubernetes
+
+  - name: Create OperatorGroup for cert-manager for Red hat OpenShift
+    kubernetes.core.k8s:
```

Comment:
You need to update this one to use the `definition` parameter (see the other k8s tasks)

Reply:
Absolutely right. Thanks for catching that!